### DISCUSSION TITLE
mutation without params

### DISCUSSION THREAD

**Discussion by kockok** [99designs/gqlgen#2181] [Q&A]
### What happened?
Just wondering if the missing `ClearUsers` method should be generated by gqlgen.
Error raised:
```
validation failed: packages.Load: /Users//schema.resolvers.go:420:67: 
cannot use &(mutationResolver literal) (value of type *mutationResolver)
 as generated.MutationResolver value in return statement: 
*mutationResolver does not implement generated.MutationResolver (missing method ClearUsers)

exit status 3
graph/resolver.go:3: running "go": exit status 1
```
### What did you expect?

I want a mutation without params, then generate a usual schema.resolvers.go without errors.

### Minimal graphql.schema and models to reproduce

if I comment out `clearUsers: Boolean!`, it works normally.

```
type Mutation {
	clearUsers: Boolean!
	sendCode(email: String!): Boolean!
	resetPw(email: String!): Boolean!
}
```

### versions
 - `go run github.com/99designs/gqlgen version`?  v0.17.5
 - `go version`? the latest

**Comment by kockok** [comment]
hello, I don't think this issue should be categorized as a discussion.

This is because if I manually setup a function like down below, it works.

```
func (r *mutationResolver) ClearUsers(ctx context.Context) (bool, error) {
...
}
```

Therefore, I reckon that this is an issue with the gqlgen code generator.

**Reply by frederikhors** [reply]
Sorry I misread your issue. Can you push a repo on GitHub please? I reopened anyway.

**Reply by kockok** [reply]
working fine with the latest release `v0.17.7`
